package/ubuntu_latext_i386_docker/Dockerfile

```diff
@@ -0,0 +1,90 @@
+FROM i386/ubuntu:18.10
+
+ARG QBDI_BUILD_TYPE=Release
+
+ENV USER docker
+ENV HOME /home/$USER
+ENV PREFIX /usr
+ENV QBDI_PLATFORM linux-X86
+
+# Get latest package list, upgrade packages, install required packages
+# and cleanup to keep container as small as possible
+RUN apt-get update \
+    && apt-get install -y \
+    bash \
+    sudo \
+    build-essential \
+    cmake \
+    g++ \
+    g++-multilib \
+    libncurses5-dev \
+    libstdc++-6-dev \
+    make \
+    pkg-config \
+    python \
+    python-dev \
+    wget \
+    zlib1g-dev \
+    vim \
+    libncurses5 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# create a user
+RUN adduser --disabled-password --gecos '' $USER && \
+    adduser $USER sudo && \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# switch to new user
+USER $USER
+
+# install QBDI compilation dependencies
+
+# git archive -o qbdi-deps.tar.gz --prefix=qbdi-deps/ HEAD deps
+ADD qbdi-deps.tar.gz $HOME/
+
+WORKDIR $HOME
+
+RUN sudo chown -R $USER:$USER .
```

Comment:
I'm having some issues when building the docker image. I get an error at this point: `OCI runtime create failed: container_linux.go:345: starting container process caused "chdir to cwd (\"/home/docker\") set in config.json failed: permission denied": unknown`. It can be fixed moving the `USER $USER` directive after this line.